alf/utils/averager.py

```diff
@@ -124,10 +124,10 @@ def __init__(self,
 class EMAverager(nn.Module):
     """Class for exponential moving average.
 
-    x_t = (1-update_rate)* x_{t-1} + update_Rate * x
-    The average is corrected by a mass as x_t / mass_t, and the mass is
+        :math:`x_t = (1-update_rate) \cdot x_{t-1} + update_Rate \cdot x`
```

Comment:
make sure to preview the result doc. It seems that this equation should be in a separate paragraph.

Reply:
updated